src/NServiceBus.TransactionalSession.AcceptanceTests.CosmosDB/NServiceBus.TransactionalSession.AcceptanceTests.CosmosDB.csproj

```diff
@@ -22,4 +21,8 @@
     <ProjectReference Include="..\NServiceBus.TransactionalSession\NServiceBus.TransactionalSession.csproj" />
     <ProjectReference Include="..\PersistenceTestHelpers\PersistenceTestHelpers.csproj" />
   </ItemGroup>
+
+  <ItemGroup>
+    <Reference Include="System.Net.Http" Condition="'$(TargetFramework)' == 'net472'" />
```

Comment:
did this missing reference cause problems somehow?

Reply:
Yeah I got build errors on Linux